### ISSUE TITLE
Could not find com.speedchecker:android-sdk:4.2.219.

### ISSUE THREAD

**Issue by zekoliman** [speedchecker/react_plugin#7]
Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:processDebugResources'.
> Could not resolve all task dependencies for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.speedchecker:android-sdk:4.2.219.
     Searched in the following locations:
       - https://oss.sonatype.org/content/repositories/snapshots/com/speedchecker/android-sdk/4.2.219/android-sdk-4.2.219.pom
       - https://repo.maven.apache.org/maven2/com/speedchecker/android-sdk/4.2.219/android-sdk-4.2.219.pom
       - file:/Users/zekican/Desktop/repolarim/newpusulam/node_modules/jsc-android/dist/com/speedchecker/android-sdk/4.2.219/android-sdk-4.2.219.pom
       - https://dl.google.com/dl/android/maven2/com/speedchecker/android-sdk/4.2.219/android-sdk-4.2.219.pom
       - https://www.jitpack.io/com/speedchecker/android-sdk/4.2.219/android-sdk-4.2.219.pom
     Required by:
         project :app > project :speedchecker_react-native-plugin

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 12s

info Run CLI with --verbose flag for more details.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by bartlenaerts** [comment]
+1 having the same issue

**Comment by bartlenaerts** [comment]
@mmh-dev , any progress on this?

**Comment by mmh-dev** [comment]
Dear Bart,

I don't work on this project anymore, for all questions related to this plugin, please write to ***@***.*** ***@***.***>

Best,
Murod

**Comment by bartlenaerts** [comment]
@mmh-dev , who do we need to trigger? It's greyed out...

**Comment by Adam014** [comment]
Having the same issue as @zekoliman

**Comment by Adam014** [comment]
Have anybody solved this or used another library for this feature?

**Comment by bartlenaerts** [comment]
Solved it!

First, add the package `expo-build-properties` (to package.json).

Second, add the MavenRepo from Speedchecker to `plugins` in `app.json`:

```
    "plugins": [
      [
        "expo-build-properties",
        {
          "android": {
            "extraMavenRepos": [
              {
                "credentials": {
                  "password": "AP3Zg8iFoiatscExgkeMHP3ur9EcsNkbxC8e6gXr8QV9PeWtiAVM3pETm8pF",
                  "username": "speedchecker"
                },
                "url": "https://maven.speedcheckerapi.com/artifactory/libs-release"
              }
            ]
          }
        }
      ],
      "expo-location",
      "expo-router",
      "expo-tracking-transparency"
    ],
```

**Comment by zekoliman** [comment]
Since the package doesn't work, I download a file manually and measure the speed @Adam014

**Comment by Adam014** [comment]
@zekoliman Alright, thanks. Not a way for me, but thanks for the heads up.

**Comment by Adam014** [comment]
> Solved it!
> 
> First, add the package `expo-build-properties` (to package.json).
> 
> Second, add the MavenRepo from Speedchecker to `plugins` in `app.json`:
> 
> ```
>     "plugins": [
>       [
>         "expo-build-properties",
>         {
>           "android": {
>             "extraMavenRepos": [
>               {
>                 "credentials": {
>                   "password": "AP3Zg8iFoiatscExgkeMHP3ur9EcsNkbxC8e6gXr8QV9PeWtiAVM3pETm8pF",
>                   "username": "speedchecker"
>                 },
>                 "url": "https://maven.speedcheckerapi.com/artifactory/libs-release"
>               }
>             ]
>           }
>         }
>       ],
>       "expo-location",
>       "expo-router",
>       "expo-tracking-transparency"
>     ],
> ```

Yep, i cant get this working with your answer. Any suggestions?

**Comment by j2pb** [comment]
+1 same issue here!

**Comment by Adam014** [comment]
@mmh-dev 

Who we need to email to about this plugin? The emails are greyed out.

**Comment by Florent75** [comment]
Hey there,
I have succeeded to make the compilation by downloading manually the android-sdk and putting it my local directory.
the local directory where the package is searching is specified at the location in the error logs : 
"file:/Users/zekican/Desktop/repolarim/newpusulam/node_modules/jsc-android/dist/com/speedchecker/android-sdk/4.2.219/android-sdk-4.2.219.pom"

You should put in this directory both the .pom file and the .aar file

This is a workaround of course

**Comment by Adam014** [comment]
Yeah this isnt fittable for us, but thanks for the heads up! Probably will create new library for this on npm when the feature we will need to implement again.

**Comment by Florent75** [comment]
Hi Adam,
I did find the solution if you are interested : 
copy past the following part in your build.gradle of your app directory : 

repositories {
    maven {
        url 'https://maven.speedcheckerapi.com/artifactory/libs-release'
        credentials {
            username = "speedchecker"
            password = "AP3Zg8iFoiatscExgkeMHP3ur9EcsNkbxC8e6gXr8QV9PeWtiAVM3pETm8pF"
        }
    }
}

**Comment by Adam014** [comment]
Thanks, will try!
